CloacaNative/IO/DataTypes/PyTextIOWrapper.cs

```diff
@@ -28,7 +28,7 @@ public class PyTextIOWrapperClass : PyClass
 
         // I don't seem to be able to add a self parameter here the argument doesn't get passed through to Injector.Inject 
         [ClassMember]
-        public static PyString readline(PyObject self)
+        public static PyString readline(PyTextIOWrapper self)
```

Comment:
Is the comment still applicable that you can't get self inserted? I would think that the PyObject argument should work? If it screws up, we should be able to generate a pretty well-isolated test for it that I can poke at in another branch.